src/main/java/com/avispl/symphony/dal/infrastructure/management/neat/pulse/common/metric/ControllerModeEnum.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *  Copyright (c) 2024 AVI-SPL, Inc. All Rights Reserved.
+ */
+
+package com.avispl.symphony.dal.infrastructure.management.neat.pulse.common.metric;
+
+/**
+ * Enum representing different types of controller mode
+ *
+ * @author Harry / Symphony Dev Team<br>
+ * Created on 5/2/2024
+ * @since 1.0.0
+ */
+public enum ControllerModeEnum {
+	SCHEDULE("schedule", "Room scheduler"),
+	CONTROLLER("controller", "Meeting room controller"),
```

Comment:
Same as CallStatusEnum, probably values should be shorther, need discuss with Nicolas